ppdet/core/workspace.py

```diff
@@ -97,6 +97,14 @@ def load_config(file_path):
         del cfg[READER_KEY]
 
     merge_config(cfg)
+
+    # NOTE: training batch size defined only in TrainReader, sychornized
+    #       batch size config to global, models can get batch size config
+    #       from global config when building model.
+    #       batch size in evaluation or inference can also be added here
+    global_config['train_batch_size'] = global_config['TrainReader'][
+        'batch_size']
```

Comment:
If ‘TrainReader’ is not in global_config, will fail

Reply:
Done, thanks!